scripts/doc_checker.py

```diff
@@ -166,6 +167,12 @@ def check_spelling_words(doc: Docstring) -> list:
         for line in doc.raw_doc.splitlines()
     ]
 
+    docstring_start_line = None
+    for idx, line in enumerate(inspect.getsourcelines(doc.code_obj)[0]):
+        if '"""' in line:
```

Comment:
while we discourage using `'''` as a docstring thing (and `black` replaces it with double quotes), I think it's better to handle both cases:
```suggestion
        if '"""' in line or "'''" in line:
```